CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ it in future.
 ### Fixed
 
 ### Changed
+* Changed the dimension of the cave: 1) Uptated dimensions of TCC8 and ECN3. 
+  2) Added a Step in ECN3. 3) Removed the coat of the absorber. 
+  4) Modified the dimension of the HA and absorber.
```

Comment:
* typo
* if you move the numbered list onto the next line (with suitable indent), it will be easier to read and parse properly as markdown
    1. as in this
    2. example
* Since we are already doing a multiline entry, we might as well link your presentation and/or the PR.